### PR TITLE
Avx loops

### PR DESCRIPTION
**Pull request by dpo**
Large speedup for multiple rhs solves:
https://gist.github.com/dpo/eb8b0e20067cbb334f7f2886168e58bc

### CODE REVIEW

**Comment by amontoison** [comment]
You have huge memory leaks for multiple rhs solves.
You cannot use a `view` on a `view` like you did without doing a copy.

**Comment by dpo** [comment]
Thanks. One of those views is clearly not needed. Should be fixed in #54.

**Comment by dpo** [comment]
Updated benchmarks: https://gist.github.com/dpo/13005601069e28e3531fa125438ab061

So I guess this isn't worth it.

**Comment by amontoison** [comment]
-> `Load Avg:  2.7900390625  2.58642578125  2.44873046875`
Do have an unused computer to run the benchmarks ? I'm scared that the results are biased.

**Comment by dpo** [comment]
https://gist.github.com/89b0d6bb727e91b5dc4ce1d3e5dda562

Basically no improvement.

File: benchmark/run_benchmarks.jl [hunk 1]
```diff
@@ -42,7 +42,9 @@ solve5_stats = Dict{Symbol,DataFrame}(:commit => commit_stats[:solve5],
 save_stats(fact_stats, "ldl_commit_vs_master_fact.jld2")
 save_stats(solve1_stats, "ldl_commit_vs_master_solve1.jld2")
 save_stats(solve5_stats, "ldl_commit_vs_master_solve5.jld2")
-save_stats(judgement_stats, "ldl_commit_vs_master_judgement.jld2")
+jldopen("ldl_commit_vs_master_judgement.jld2", "w") do file
+  file["jstats"] = judgement_stats
+end
```
Comment (amontoison) [inline benchmark/run_benchmarks.jl:47]: Can you merge this even if Avx doesn't give any improvement ?

**Comment by amontoison** [comment]
Thanks for the benchmarks!

File: benchmark/run_benchmarks.jl (hunk 1 above)
Comment (dpo) [reply benchmark/run_benchmarks.jl:47]: It's in #58.

**Pull request closed without merging**